package.json

```diff
@@ -39,11 +40,15 @@
         "@eslint/js": "^9.8.0",
         "@lwc/eslint-plugin-lwc-internal": "link:./scripts/eslint-plugin",
         "@lwc/test-utils-lwc-internals": "link:./scripts/test-utils",
+        "@nx/js": "19.5.6",
         "@rollup/plugin-commonjs": "^26.0.1",
         "@rollup/plugin-inject": "^5.0.5",
         "@rollup/plugin-node-resolve": "^15.2.3",
         "@rollup/plugin-replace": "^5.0.7",
         "@rollup/plugin-typescript": "^11.1.6",
+        "@swc-node/register": "~1.9.1",
+        "@swc/core": "~1.5.7",
+        "@swc/helpers": "~0.5.11",
```

Comment:
I find it bizarre that Nx wants these things, but 🤷

Reply:
When I ran `nx release version` it told me "hey you gotta run `nx add @nx/js` first", so I did. These are for TypeScript support, but it's unclear whether nx needs them or just thinks we might want them.